src/redux/configureStore.js

```diff
@@ -0,0 +1,11 @@
+import { combineReducers, createStore } from 'redux';
```

Comment:
- [ ] After you have installed the redux-toolkit, it will be easy to configure the store as there is a function called `configureStore` that will do it for you 😮 
```
Import configureStore from @reduxjs/toolkit.
```
You will use this function to create the reducer for books and categories. 😃 
Please read this [documentation](https://redux-toolkit.js.org/tutorials/quick-start)